cylc/flow/cfgspec/workflow.py

```diff
@@ -283,28 +327,53 @@ def get_script_common_text(this: str, example: Optional[str] = None):
             one here, it will only be used for inferring timezone-less cycle
             points, while dumping will use the one from the cycle point format.
 
-            .. note::
+            .. caution::
 
                It is not recommended to write the time zone with a ":"
                (e.g. ``+05:30``), given that the time zone is used as part of
                task output filenames.
-
-            .. versionchanged:: 7.8.9/7.9.4
-
-               The value set here now persists over reloads/restarts after a
-               system time zone change.
         ''')
 
-        with Conf('main loop'):  # noqa: SIM117 (keep same format)
+        with Conf(   # noqa: SIM117 (keep same format)
+            'main loop',
+            desc='''
+                Allows the specification of main loop plugins for Cylc.
+
+                For a list of built in plugins see :ref:`BuiltInPlugins`.
```

Comment:
> Also, might be worth linking to the the main section, rather than specifically the list of built-in plugins.

I'm not sure that there's a great deal more in the docs _to_ be referenced.

Reply:
I just think users might be unlikely to click a link to the built-in plugins, and might appreciate a more general
```
See :ref:`Main Loop Plugins <MainLoopPlugins>` for details and a list of built-in plugins
```
(Would need to create a `MainLoopPlugins` target or whatever it's called in the main loop rst file in cylc-doc, I think)

Reply:
```suggestion
                For a list of built in plugins see :ref:`Main Loop Plugins <BuiltInPlugins>`.
```